src/database/migrations/1731934120969-update-category-amenity-table.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
Why are you dropping the icon column in amenities and categories? what if the name is convenient to show users?  or your image on your storage? like house-icon in your storage and on the frontend we should also display the name of amenity or category

Reply:
but didn't we agree that we will handle icons on the frontend with matching category/amenity names to icons? When will we be showing the name of the icon to users? I agree it is convenient to have icon names in db but right now it is not needed. How about I make the icon field Nullable for now instead of deleting it?